chat-widget/src/components/livechatwidget/interfaces/ILiveChatWidgetProps.ts

```diff
@@ -64,4 +65,5 @@ export interface ILiveChatWidgetProps {
     allowSdkChatSupport?: boolean; // to avoid any performance impact
     // eslint-disable-next-line @typescript-eslint/no-explicit-any
     initialCustomContext?: any;
+    draggableChatWidgetProps?: IDraggableChatWidgetProps;
```

Comment:
Please add a table entry and create a PR into this branch https://github.com/charliewang95/omnichannel-chat-widget/blob/dev/docs/customizations/livechatwidget.md#:~:text=initialCustomContext